packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -108,11 +108,22 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
    * Renders a pivot link
    */
   private _renderLink(link: IPivotItemProps) {
-    const { itemKey, itemCount } = link;
+    const { itemKey, itemCount, itemIcon, linkText } = link;
     let { id } = this.state;
-    let countText;
+
+    let count;
     if (itemCount !== undefined && this.props.linkFormat !== PivotLinkFormat.tabs) {
-      countText = <span className='ms-Pivot-count'>({ itemCount })</span>;
+      count = <span className='ms-Pivot-count'>({ itemCount })</span>;
+    }
+
+    let icon: JSX.Element;
+    if (itemIcon !== undefined && this.props.linkFormat !== PivotLinkFormat.tabs) {
```

Comment:
I just followed the existing code that doesn't allow `count` to be shown when in tabs mode, you think we should allow both?